omero/sysadmins/unix/walkthrough/walkthrough_centos7.sh

```diff
@@ -66,32 +64,48 @@ su - postgres -c "createdb -E UTF8 -O '$OMERO_DB_USER' '$OMERO_DB_NAME'"
 psql -P pager=off -h localhost -U "$OMERO_DB_USER" -l
 #end-step03
 
-#start-step04: As the omero system user, install the OMERO.server
+#start-step03bis: As the omero system user, create a virtual env and install dependenciees
+# Create a virtual env and activate it
+VIRTUALENV=${VIRTUALENV:-/home/omero/omeroenv}
+python3 -mvenv $VIRTUALENV
+. $VIRTUALENV/bin/activate
+
+# Install the Ice Python binding
+pip3 install https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.2.1/zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl
+#end-step03bis
+
+#start-step04: As the omero system user, install the omero-py OMERO.server
 #start-copy-omeroscript
 cp settings.env settings-web.env step04_all_omero.sh setup_omero_db.sh ~omero 
 #end-copy-omeroscript
 #start-release-ice36
 cd ~omero
-SERVER=https://downloads.openmicroscopy.org/latest/omero5.5/server-ice36.zip
+#SERVER=https://downloads.openmicroscopy.org/latest/omero5.6/server-ice36.zip
+SERVER=https://downloads.openmicroscopy.org/omero/5.6.0-m2/artifacts/OMERO.server-5.6.0-m2-ice36-b126.zip
 wget -q $SERVER -O OMERO.server-ice36.zip
 unzip -q OMERO.server*
 #end-release-ice36
 ln -s OMERO.server-*/ OMERO.server
-OMERO.server/bin/omero config set omero.data.dir "$OMERO_DATA_DIR"
-OMERO.server/bin/omero config set omero.db.name "$OMERO_DB_NAME"
-OMERO.server/bin/omero config set omero.db.user "$OMERO_DB_USER"
-OMERO.server/bin/omero config set omero.db.pass "$OMERO_DB_PASS"
-OMERO.server/bin/omero db script -f OMERO.server/db.sql --password "$OMERO_ROOT_PASS"
+# set OMERODIR
+export OMERODIR=OMERO.server
```

Comment:
probably safer to use the full absolute path

Reply:
this has been changed in omero-install